client/changelist/change.go

```diff
@@ -20,7 +20,7 @@ const (
 // all changes in Snapshot and Timestamp are programmatically
 // generated base on Root and Targets changes.
 const (
-	TypeRootRole          = "role"
+	TypeRootRole          = "root"
```

Comment:
@ecordell is correct on the use case. Some more context, it's used specifically to describe a change type that modifies a base role, as opposed to a delegation role. Probably `TypeBaseRole` makes the most sense, to make it clear it's not to be used for delegation roles.